srv/components/misc_pkgs/rabbitmq/config/base.sls

```diff
@@ -32,6 +32,16 @@ Set RabbitMQ environment:
     - watch_in:
       - Start RabbitMQ service
 
+Set RabbitMQ configuration:
```

Comment:
this is nice, replacing just the config line we need, wasn't aware this was possible.